notion/README.md

```diff
@@ -0,0 +1,22 @@
+# Agent Check: Notion
+
+## Overview
+
+Notion is a Connected Workspace. It's where modern teams create and share docs, take notes, manage projects, and organize knowledge-all in one place. Install Notion's integration with Datadog to manage and monitor your workspace activity in their Datadog Cloud SIEM. You can import your workspace's audit logs for real-time monitoring, alerting, and analysis. From there, you can detect and investigate potential security issues, suspicious behavior, and troubleshoot access with confidence and ease.
+
+## Setup
+
+In Notion navigate to Setting & Members > Connections > Workspace Connections > +Add Connection > Datadog. 
+
+This will start the OAuth flow with Datadog. Once you've successfully authenticated, you will be redirected back to Notion, an integration tile will be created in the workspace connections view. 
```

Comment:
Users should be directed to kick off authorization from the Datadog tile. If a user kicks off authorization directly from your platform, this site parameter will not be sent, and the user will be prompted to select their site on the Datadog authorize page. Please update the instructions to reflect the above!